src/Microsoft.Graph.Core/Exceptions/ServiceException.cs

```diff
@@ -53,6 +71,11 @@ public ServiceException(Error error, Exception innerException = null)
         /// </summary>
         public System.Net.HttpStatusCode StatusCode { get; }
 
+        /// <summary>
+        /// Provide the raw JSON response body.
+        /// </summary>
+        public string RawResponseBody { get; }
```

Comment:
If we have Error, which should be a strongly typed version of the body, what is the reason for also keeping the rawResponseBody around too?

Reply:
A customer requested this, and we added this to our todo list. This was from this feature request: https://github.com/microsoftgraph/msgraph-sdk-dotnet/issues/316.

Fair enough question since it is all available in the Error object.

Reply:
I think the main point from that issue is that there were scenarios where we were not returning an error object at all.  e.g. An action that didn't have a return type.  I think returning a raw response was just the workaround solution

Reply:
We have a scenario where we swallow the error response returned in the response body when there is a ErrorResponse deserialization exception. We should provide the response body in that scenario.